### PR TITLE
Patched profile urls #2

### LINKED ISSUE #2: Profile URLs broken
In the provider YAMLs, we have a `profileBaseUrl` set, which we currently set to the full URL of a provider's homepage (e.g. https://qjcr.org.uk)

This doesn't surface correctly on the website. Clicking on an organisation link e.g.
image

takes you to `http://camb.votehttps//qjcr.org.uk/`. The same bad link is visible in the ballot creation page among others

Two solutions:
1. Fix these links in the GUI
2. Change the YAMLs to set `profileBaseUrl=/qjcr` (for example)

### PR DESCRIPTION
**Pull request by lewis-wf**
Bodge for #2

### CODE REVIEW

File: qjcr.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 qjcr:
   id: qjcr
   name: Queens' College JCR
-  baseUrl: "/"
+  baseUrl: "/qjcr."
```
Comment (EmilyBoarer) [inline qjcr.yaml]: should end in slash not dot?

**Comment by EmilyBoarer** [comment]
qjcr url incorrect?

File: qjcr.yaml (hunk 1 above)
Comment (lewis-wf) [reply qjcr.yaml]: Good catch - fixed https://github.com/camvote/bob-providers/pull/3/commits/d4117e34b27551f473f4be0f4fda1b0e6fa31a3d

**Review by mxbi — APPROVED** [review]
Looks good - I'll merge. Note that I don't think this will affect links on currently running elections as the metadata gets copied over every time.

Thanks!

**Pull request merged**